tests/e2e/env/bin/e2e-test-integration.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+const { spawnSync } = require( 'child_process' );
+const program = require( 'commander' );
+const path = require( 'path' );
+const fs = require( 'fs' );
+const getAppPath = require( '../utils/app-root' );
+
+program
+	.usage( '<file ...> [options]' )
+	.option( '--dev', 'Development mode' )
+	.parse( process.argv );
+
+const appPath = getAppPath();
+
+const nodeConfigDirs = [
+	path.resolve( __dirname, '../config' ),
+];
+
+if ( appPath ) {
+	nodeConfigDirs.unshift(
+		path.resolve( appPath, 'tests/e2e/config' )
+	);
+}
+
+const testEnvVars = {
+	NODE_ENV: 'test-e2e',
```

Comment:
I expect we will iterate on the component as part of the process of implementing it in `woocommerce-admin`. In that process, I hope to remove some of the hardcoded configuration (eg. requiring port 8084).

This is already a big PR & I'd prefer to get it into master so that we can do smaller changes in individual PRs.

Reply:
I'll never say no to dogfooding 😁

The only thing to note is that this _particular_ change will also require they rename their file, so depending on how we version the package, that may be problematic.

Reply:
We could change the package version to be `0.1` to start then update it to `1.0` once we having the configuration coordinated between the package and a consumer (or two).

Reply:
I think that would be a good way to iterate on this in smaller PRs without worrying too much about breaking anything.